plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/internal/ResultSetMessages.properties

```diff
@@ -270,6 +270,8 @@ pref_page_database_resultsets_label_use_sql_tip = Modify source SQL query to scr
 pref_page_database_resultsets_group_string = Strings
 pref_page_database_resultsets_checkbox_string_use_editor = Open in separate editor
 pref_page_database_resultsets_checkbox_string_use_editor_tip = Opens separate string editor instead of dialog
+pref_page_database_resultsets_label_replace_nulls_on_default = Replace all nulls insert in default values
+pref_page_database_resultsets_label_replace_nulls_on_default_tip = Use "DEFAULT VALUES" insert, when in the row only zero values. Database specific setting.
```

Comment:
```suggestion
pref_page_database_resultsets_label_replace_nulls_on_default = Insert default values instead of all null values
pref_page_database_resultsets_label_replace_nulls_on_default_tip = Use "DEFAULT VALUES" insert when the row does not contain any values.\nThis is database-specific setting and may not work in some environments.
```